package/autoyast2.spec

```diff
@@ -226,11 +226,7 @@ done
 %{yast_moduledir}/AutoinstClone.rb
 %dir %{yast_yncludedir}/autoinstall
 %{yast_yncludedir}/autoinstall/io.rb
-%{yast_yncludedir}/autoinstall/AdvancedPartitionDialog.rb
-%{yast_yncludedir}/autoinstall/DriveDialog.rb
-%{yast_yncludedir}/autoinstall/PartitionDialog.rb
-%{yast_yncludedir}/autoinstall/StorageDialog.rb
-%{yast_yncludedir}/autoinstall/VolgroupDialog.rb
+%{yast_yncludedir}/autoinstall/autoinst_dialogs.rb
```

Comment:
I don't think such file exists

Reply:
An this line was not there in #614, I don't know where it comes from.

Reply:
Maybe an error during some conflict resolution?

Reply:
Yes, it looks like an error/overlook during a conflict resolution.

Thanks!